draft-ietf-quic-manageability.md

```diff
@@ -905,6 +905,119 @@ need to be flow based.  However, 5-tuple (plus eventually connection ID if
 present) matching is still beneficial for QoS given all packets are handled by
 the same congestion controller.
 
+## QUIC and Network Address Translation (NAT)
+
+Network Address Translators (NATs) are a widely deployed means of multiplexing
+multiple private IP addresses over scarce IPv4 public address space by replacing
+those addresses and using ports to distinguish those connections. The new
+address can also guarantee that packets move through a proxy throughout the life
+of a connection, so that the connection can continue with the required state at
+that proxy.
+
+This section uses the colloquial term NAT to mean NAPT (section 2.2 of
+{{?RFC3022}}), which overloads several IP addresses to one IP address or to an
+IP address pool, as commonly deployed in carrier-grade NATs or residential NATs.
+QUIC Connection IDs are opaque byte fields that are expressed consistently
+across all QUIC versions {{QUIC-INVARIANTS}}. This feature may appear to present
+opportunities to optimize NAT port usage and simplify the work of the QUIC
+server. In fact, NAT behavior that relies on CID may instead cause connection
+failure when endpoints change Connection ID, and disable important protocol
+security features.
+
+The remainder of this section explains how QUIC supports NATs better than other
+connection-oriented protocols, why NAT use of Connection ID might appear
+attractive, and how NAT use of CID can create serious problems for the
+endpoints.
+
+The conclusion of this section is that NATs should retain their existing
```

Comment:
I'd maybe move this up, or summarize it in the first para. This section should be very clear about the "don't do it"ness of messing with CIDs.